src/v1/models/app/post/index.js

```diff
@@ -48,19 +48,43 @@ export function postModel() {
 
 const Post = postModel();
 
-export function createPost(data) {
-  return new Post(data)
+export async function createPost(data, file = null) {
```

Comment:
Eu num entendo muito do porque, mas há necessidade de introduzir null no código?
Não estou dizendo que seja problema, mas grande partes das vezes introduzir null no código traz uma maldição que sempre que for avaliar esta variavel ter que fazer `if (file !== null)`, seria possível eliminar o uso de null?